apps/web/eslint.config.js

```diff
@@ -1,4 +1,42 @@
 import { nextJsConfig } from "@repo/eslint-config/next-js";
+import vitest from "eslint-plugin-vitest";
+import testingLibrary from "eslint-plugin-testing-library";
+import jestDom from "eslint-plugin-jest-dom";
 
-/** @type {import("eslint").Linter.Config} */
-export default nextJsConfig;
+/** @type {import("eslint").Linter.FlatConfig[]} */
+export default [
+  ...nextJsConfig,
+  {
```

Comment:
all these configs should be sharable

Reply:
Since we use a monorepo, we have a workspace package to store eslint configs.

Organize and move all the ESLint setup related to tests into `/packages/eslint-config`. The name could be "unit-tests" and then use the ready config in `apps/web/eslint.config.js`